developer_manual/basics/front-end/l10n.rst

```diff
@@ -256,7 +256,7 @@ For vue js code, see the javascript section.
 Adding translations
 -------------------
 
-Nextcloud's translation system is powered by `Transifex <https://www.transifex.com/nextcloud/>`_. To start translating sign up and enter a group. If your community app should be translated by the `Nextcloud community on Transifex <https://www.transifex.com/nextcloud/nextcloud/dashboard/>`_ just follow the setup section below.
+Nextcloud's translation system is powered by `Transifex <https://explore.transifex.com/nextcloud/>`_. To start translating sign up and enter a group. If your community app should be translated by the `Nextcloud community on Transifex <https://explore.transifex.com/nextcloud/>`_ just follow the setup section below.
```

Comment:
the dashboard I used to have is now on:
https://app.transifex.com/nextcloud/

Reply:
Only works when logged in:

app | explore
---|---
![Screenshot 2024-10-11 at 14-24-15 Log in · Transifex](https://github.com/user-attachments/assets/c97c0758-98a2-43de-831e-42a699f5aa90)|![Screenshot 2024-10-11 at 14-24-21 Nextcloud localization Transifex](https://github.com/user-attachments/assets/0851da5f-da7f-4f71-8266-a0af6778e783)

Reply:
ok, good point, new users probably have no account...